packages/tasks/src/hardware.ts

```diff
@@ -0,0 +1,305 @@
+/**
+ * Biden AI Executive Order
+ * https://www.whitehouse.gov/briefing-room/presidential-actions/2023/10/30/executive-order-on-the-safe-secure-and-trustworthy-development-and-use-of-artificial-intelligence/
+ */
+export const TFLOPS_THRESHOLD_WHITE_HOUSE_MODEL_TRAINING_TOTAL = 10 ** 14;
+export const TFLOPS_THRESHOLD_WHITE_HOUSE_MODEL_TRAINING_TOTAL_BIOLOGY = 10 ** 11;
+export const TFLOPS_THRESHOLD_WHITE_HOUSE_CLUSTER = 10 ** 8;
+
+/**
+ * EU AI Act
+ * https://ec.europa.eu/commission/presscorner/detail/en/qanda_21_1683
+ */
+export const TFLOPS_THRESHOLD_EU_AI_ACT_MODEL_TRAINING_TOTAL = 10 ** 13;
+
+export interface HardwareSpec {
+	/**
+	 * Approximate value, in FP32.
+	 * This is only approximate/theoretical and shouldn't be taken too seriously.
+	 * Currently the CPU values are from cpu-monkey.com
+	 * while the GPU values are from techpowerup.com
+	 *
+	 * Note to reviewers: I got fed up with data entry,
+	 * and HuggingChat running Llama3 with Web search was failing a bit,
+	 * so some of those values are kinda random. Forgive me and please feel free to improve.
+	 */
+	tflops: number;
+	/**
+	 * If an array is specified, options of memory size (can be VRAM, unified RAM)
+	 * e.g. an A100 exists in 40 or 80 GB.
+	 */
+	memory?: number[];
+}
+
+export const DEFAULT_MEMORY_OPTIONS = [8, 16, 24, 32, 40, 48, 64, 80];
```

Comment:
Also 12?

Reply:
What is it used for? Missing from GPU/CPU: 10, 12, 18, 20, 36, 96, 128, 192

Reply:
it's used as a default list to pick from when `SKUS` does not define a `memory` list of options. so mostly for CPUs!

Reply:
in light of this new info, what new list items do you suggest? (i think it's ok if it's not 100% exhaustive, people can choose the closest one)

Reply:
eff64a62729620bc089a2ca4f2d07a74f7b72e1f